### PR TITLE
Inspector issues view update - make more actionable and improve clarity and phrasing

### PR DESCRIPTION
**Pull request by 1212spark**
- Pull up "Fixing an issue found in Inspector" section and flesh out further
- Make language more actionable throughout; e.g. move from "It's helpful to sort by" to "Sort by"
- Proofread for improved clarity throughout the doc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206555865653134

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…/inspector-issues-view**](https://docs-mzyi6ig5t.avo-dev.app/inspector/inspector-issues-view) |
|:-:|
|<a href="https://docs-mzyi6ig5t.avo-dev.app/inspector/inspector-issues-view">Screenshot of /inspector/inspector-issues-view</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/zsGjLNDX7TapbLZXnAYpI8vTdXdXazjr5RHMa6BIcU8RYccfrQdzMZ7nTqNVfydalbO9j7g_iFMczjt0yHkvcQ.png">(view full size)</a> |







Commit 48afcf14bd1f0868bdcfaf694740666268b98aa6 (https://docs-mzyi6ig5t.avo-dev.app).

**Comment by 1212spark** [comment]
@klaraavo and/or @spickerill - do y'all have write access (thereby the ability to approve if and when this looks good)?

**Review by logason — APPROVED** [review]
LGTM!

**Pull request merged**
